Short-Answer/Algorithms_Answers.md

```diff
@@ -1,15 +1,41 @@
-#### Please add your answers to the ***Analysis of  Algorithms*** exercises here.
+#### Please add your answers to the **_Analysis of Algorithms_** exercises here.
+
+<!-- Give an analysis of the running time of each snippet of
+pseudocode with respect to the input size n of each of the following: -->
 
 ## Exercise I
 
-a)
+a) a = 0 # single operation runtime O(1)
+while (a < n _ n _ n): # for loop O(n) no nested loops
+a = a + n \* n #
+
+# I believe this is an O(n) runtime
+
+b) sum = 0 # single operation runtime O(1)
+for i in range(n): # for loop O(n) with a nested for loop
+j = 1
+while j < n: # nested for loop so we multiple log(n) ( logn \* O(n) = log(n)) )
+j \*= 2
+sum += 1
 
+# I believe this is an log(n) runtime (formatting gets messed up when saving)
 
-b)
+c) def bunnyEars(bunnies):
+if bunnies == 0: # O(n)
+return 0
 
+      return 2 + bunnyEars(bunnies-1) # single operation runtime O(1)
 
-c)
+# I believe this is an O(n) runtime
 
 ## Exercise II
```

Comment:
Congrats for scoring 3 out of 3 points for Exercise II, @MelvinPeepers! 🔥 
<img width="1280" alt="exercise_ii" src="https://user-images.githubusercontent.com/16110931/76693790-6e709200-6641-11ea-95dd-fbcc14751840.png">
<img width="881" alt="Screen Shot 2020-03-14 at 10 14 52 PM" src="https://user-images.githubusercontent.com/16110931/76693791-716b8280-6641-11ea-87ff-927cb9fc0d43.png">